demos/sample_azure_iot/sample_azure_iot.c

```diff
@@ -78,26 +81,72 @@
 /**
  * @brief The Telemetry message published in this example.
  */
-#define sampleazureiotMESSAGE                                 "Hello World : %d !"
+// #define sampleazureiotMESSAGE                                 "Hello World : %d !"
+// #define sampleazureiotMESSAGE                                 "{\"temperature\":%f,\"humidity\":%f,\"co2\":%f}"
+
+/**
+ * @brief Telemetry Values
+ */
+// Primary objects
+#define sampleazureiotTELEMETRY_OBJECT_SKID                       ( "skid" )
+#define sampleazureiotTELEMETRY_OBJECT_UNITS                      ( "units" )
+#define sampleazureiotTELEMETRY_OBJECT_EXTERNAL                   ( "environment" )
+#define sampleazureiotTELEMETRY_OBJECT_STATE                      ( "state" )
+#define sampleazureiotTELEMETRY_OBJECT_ERROR_STATE                ( "error-state" )
+// Secondary
+#define sampleazureiotTELEMETRY_OBJECT_UNIT1                      ( "unit1" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT2                      ( "unit2" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT3                      ( "unit3" )
+#define sampleazureiotTELEMETRY_OBJECT_PRESSURE                   ( "pressure" )
+#define sampleazureiotTELEMETRY_OBJECT_TEMPERATURE                ( "temperature" )
+
+// Telemetry
+#define sampleazureiotTELEMETRY_PRESSURE1                         ( "pressure1" )
+#define sampleazureiotTELEMETRY_PRESSURE2                         ( "pressure2" )
+#define sampleazureiotTELEMETRY_PRESSURE3                         ( "pressure3" )
+#define sampleazureiotTELEMETRY_MASS_FLOW                         ( "massflow" )
```

Comment:
we may need 2 massflow values.

Reply:
Sure, we will make changes as we need.
This is as per the initial inputs. Let us meet again, including Viktor, after a few days once we are sure about the whole list.